docs/db.rst

```diff
@@ -0,0 +1,37 @@
+========================
+Freshmaker database
+========================
+
+Freshmaker as an SQL toolkit uses SQLAlchemy extension for `Flask <https://flask-sqlalchemy.palletsprojects.com/en/2.x/>`_. So whenever you want to add some columns or event tables you should at least check ``models.py`` file. Every class that has a **FreshmakerBase** as a parent is actually a table with it's own columns, relationships etc.
```

Comment:
"So whenever you want to add some columns or event tables" -- not sure why "event tables", did you mean `new tables`?

Reply:
s/a table with it's own columns/a table with its own columns/g

Reply:
@gnaponie I suggest you use the "Insert Suggestion" feature of GitHub instead when suggesting changes like this  :smile:

Reply:
@mprahl I guess I'm getting old :D